db/seeds.rb

```diff
@@ -5,3 +5,28 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+#
+require 'faker'
+
+password = 'password'
+
+user = User.create(
+  username: Faker::Internet.user_name,
+  password: password
+)
+
+billing_infos = BillingInfo.create(
+  last4: Faker::Number.number(4),
+  country: Faker::Address.country,
+  brand: Faker::Business.credit_card_type,
+  user_id: User.last.id { |users| (p users.id)}
```

Comment:
Hey, @SemyzSerhiy I would prefer to use `user` variable to set `user_id` here
`user_id: user.id`